add initial content to data browser workspace

davidsbailey opened the pull request:
spec: https://docs.google.com/document/d/1UfBgHCisr_8jSt8zHFgJL-XrlLthTrXJQZJwoedGYg4/edit#heading=h.4ikfybcr5h8z 

screenshot:
<img width="1177" alt="screen shot 2016-06-28 at 10 40 04 am" src="https://cloud.githubusercontent.com/assets/8001765/16426361/92efe988-3d1d-11e6-80c7-57f09eea8d64.png">

davidsbailey commented:
Hey @poorvasingal , here's a first crack at implementing the first screen in the data browser UI. Please let me know if you have any initial comments! We can do a more detailed pass later, but any mistakes present now are likely to get replicated as I add more stuff to this :)

poorvasingal commented:
Nice @davidsbailey!  The only thing I can think of to tweak is the border around the "table name" edit field.  Can we make it more like a dark gray 1 pixel border w/ rounded corners?  Kind of like the screen selector in the phone frame on the left?

Bjvanminnen commented on apps/src/templates/DataWorkspace.jsx:
```diff
@@ -1,16 +1,157 @@
 import React from 'react';
+import PaneHeader from './PaneHeader';
+const PaneSection = PaneHeader.PaneSection;
```
i think this can be `import PaneHeader, { PaneSection } from './PaneHeader';

Bjvanminnen commented on apps/src/templates/DataWorkspace.jsx:
```diff
@@ -1,16 +1,157 @@
+           </table>
+          </div>
+        </div>
+>>>>>>> add initial content to dataWorksapce
```
leftover from merge presumably?

Bjvanminnen commented on apps/src/templates/DataWorkspace.jsx:
```diff
@@ -1,16 +1,157 @@
+
+export default connect(state => ({
+  localeDirection: state.pageConstants.localeDirection,
+  isRunning: !!state.runState.isRunning
```
could add these to propTypes

davidsbailey replied:
looks possible to enforce this with eslint: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md

Bjvanminnen replied:
Oooh, that would be sweet. The other things I'd been considering trying to build (if such a thing doesn't exist already) is the ability to auto-populate propTypes based on info provided to connect, so that I don't end up having to list all these props in two places.

Bjvanminnen commented:
nice work! lgtm

davidsbailey commented:
Thanks @poorvasingal and @brent! I've addressed all feedback. will merge once tests pass
<img width="863" alt="screen shot 2016-06-28 at 12 10 09 pm" src="https://cloud.githubusercontent.com/assets/8001765/16429098/ebef97fc-3d29-11e6-8827-02f80558caa5.png">
